jupyterbook/content/code_gallery/data_access_notebooks/2017-01-18-siphon-explore-thredds.ipynb

```diff
@@ -11,7 +11,9 @@
     "[Siphon](http://siphon.readthedocs.io/en/latest/) is a Python module for accessing data hosted on a THREDDS data server.\n",
```

Comment:
<p>I get an error that "Firefox can't open this page".</p><p> </p><p>Probably something on my end but I figured I'd mention it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ioos/ioos_code_lab/pull/66/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
Nope. Probably a redirect issue. The new page is https://unidata.github.io/siphon/latest, I'l update that.

Reply:
![image](https://user-images.githubusercontent.com/8480023/154108945-a454b59f-6ac8-45bf-a371-1f3062889f13.png)

Still having an issue with `IFrame` 🤔

Reply:
Looks like the server no longer allow us to use IFrame:

`thredds.cencoos.org refused to connect.`

I'll use a screenshot instead :-(